data/block_export_data.go

```diff
@@ -69,38 +67,46 @@ func (blkData *BlockData) UpdateFromEncodedBlockCertificate(input *rpcs.EncodedB
 
 // MakeBlockDataFromEncodedBlockCertificate makes BlockData from rpcs.EncodedBlockCert
 func MakeBlockDataFromEncodedBlockCertificate(input *rpcs.EncodedBlockCert) BlockData {
```

Comment:
Can we get rid of this now? What else needs the encoded block cert converted after this PR?

Reply:
block_processor still uses this. we can remove it when removing the block_processor which is one of the last steps.